src/main/kotlin/app/revanced/patches/youtube/misc/videobuffer/fingerprints/MaxBufferFingerprint.kt

```diff
@@ -0,0 +1,32 @@
+package app.revanced.patches.youtube.misc.videobuffer.fingerprints
+
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.extensions.or
+import app.revanced.patcher.fingerprint.method.annotation.DirectPatternScanMethod
+import app.revanced.patcher.fingerprint.method.annotation.MatchingMethod
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import app.revanced.patches.youtube.misc.videobuffer.annotations.CustomVideoBufferCompatibility
+import org.jf.dexlib2.AccessFlags
+import org.jf.dexlib2.Opcode
+import org.jf.dexlib2.iface.instruction.NarrowLiteralInstruction
+
+@Name("maxbuffer-fingerprint")
+@MatchingMethod(
+    "Lcom/google/android/libraries/youtube/innertube/model/media/PlayerConfigModel;", "r"
+)
+@DirectPatternScanMethod
+@CustomVideoBufferCompatibility
+@Version("0.0.1")
+object MaxBufferFingerprint : MethodFingerprint(
+    "I", AccessFlags.PUBLIC or AccessFlags.FINAL, listOf(),
+    listOf(Opcode.SGET_OBJECT, Opcode.IGET, Opcode.IF_EQZ, Opcode.RETURN),
+    null,
+    customFingerprint = { methodDef ->
+        methodDef.definingClass.equals("Lcom/google/android/libraries/youtube/innertube/model/media/PlayerConfigModel;")
+                && methodDef.implementation!!.instructions.any {
+            ((it as? NarrowLiteralInstruction)?.narrowLiteral == 120000)
+                    && methodDef.name.equals("r")
```

Comment:
Yeah no, this is not going to work.

Reply:
it does work. And it does work from vanced to current version. There is NO other way of doing it sadly. except removing the 120000 check. Feel free to look for a better way and tell me when you have one xD

Reply:
Which class, method and APK version?

Reply:
> Which class, method and APK version?

APK/version: https://github.com/revanced/revanced-patches/pull/137/files#diff-b594e6748d60559d509da318abdc7b2dd3a85d04277da1142548eca5e5d8dcb8R8
Class: https://github.com/revanced/revanced-patches/pull/137/files#diff-4d2bdccb729ae99fc177a05695c5cc465bbbd20e0db521008c6db22e2a2c9da8R26

Method matching this: https://github.com/revanced/revanced-patches/pull/137/files#diff-4d2bdccb729ae99fc177a05695c5cc465bbbd20e0db521008c6db22e2a2c9da8R28-R29

Reply:
> Which class, method and APK version?

You can see it all in the code